deployment/bahisweb/Dockerfile

```diff
@@ -76,4 +76,4 @@ ENV DJANGO_DEBUG="False"
 
 #CMD ["./uwsgi", "bahis.ini"]
 #we need 10s delay to give databases time to initialise
-CMD sleep 10 && python /${SRC_DIR}/kobocat/manage.py collectstatic --noinput && ./celeryd_bahis start && ./uwsgi bahis.ini
+CMD sleep 10 && python /${SRC_DIR}/kobocat/manage.py collectstatic --noinput && python /${SRC_DIR}/kobocat/manage.py migrate && ./celeryd_bahis start && ./uwsgi bahis.ini
```

Comment:
Django migration creates lots of duplicate tables. We can stick to no migration at the startup container. if someone makes new model then he will be responsible for doing the migration for that particular app.

Reply:
Okay then, let's close this PR without merging.

Reply:
Changed to `migrate bh_module` is that what you meant?

Reply:
I meant we do not need to run any kind of migration at the startup of the container. It is only one-time job. If someone makes any changes in the model then we can manually migrate it from the terminal of that container.

Reply:
Ah, got ya. Let's close this PR without merging then.